Creates a command voidsuit to replace the pilot voidsuit in the bridge EVA

Flying-loulou opened the pull request:
:cl: Flying_loulou
rscadd: Adds a new command voidsuit type for bridge officers and the SEA.
maptweak: Replaces the pilot voidsuit in the bridge EVA storage by the command voidsuit.
/:cl:

Creates a command voidsuit for the bridge officers (/the SEA) and replaces the pilot voidsuit in the bridge EVA storage by said command voidsuit.

As a side note this is based on the exploration/medical voidsuits sprites, which I slightly modified (recoloration, shoulder stripes slightly larger, and so on). The suit has the same caracteristics than the medical one.

Some Pics :
![Command_void-1](https://user-images.githubusercontent.com/49248697/56249564-7ed46180-60ac-11e9-89f0-3998d23d25c9.JPG)
![Command_void-2](https://user-images.githubusercontent.com/49248697/56249565-7f6cf800-60ac-11e9-9a45-4f610b47b5fa.JPG)
![Command_void-3](https://user-images.githubusercontent.com/49248697/56249559-7ed46180-60ac-11e9-8dd6-ef82da09e0bc.JPG)
![Command_void-4](https://user-images.githubusercontent.com/49248697/56249560-7ed46180-60ac-11e9-8021-13b0b585476c.JPG)
![Command_void-5](https://user-images.githubusercontent.com/49248697/56249561-7ed46180-60ac-11e9-8efb-400114b7734f.JPG)
![Command_void-6](https://user-images.githubusercontent.com/49248697/56249562-7ed46180-60ac-11e9-93ef-2ac668c26c5d.JPG)

Flying-loulou committed:
Creates a "command voidsuit" to replaces the pilot voidsuit in the bridge EVA storage.

babydoll commented on code/modules/clothing/spacesuits/void/station.dm:
```diff
@@ -320,3 +320,28 @@
+
+
+	//Command
+/obj/item/clothing/head/helmet/space/void/command
```
Both of these should be in a torch map folder

babydoll commented on code/game/machinery/suit_storage_unit.dm:
```diff
@@ -135,6 +135,16 @@
 	helmet_type = /obj/item/clothing/head/helmet/space/void/security/alt
 	mask_type = /obj/item/clothing/mask/breath
 
+/obj/machinery/suit_storage_unit/command
```
This should also be in the torch map folder

babydoll commented:
The icons should also be moved to the relevant files. alternatively you can scrub any references to torch-specific concepts and leave them where they are

Flying-loulou commented:
> 
> 
>  alternatively you can scrub any references to torch-specific concepts and leave them where they are

I'd rather do that, but I don't see any Torch-specific concept I used, aside the explorer suit which is, itself, in those files already

babydoll commented:
the explorer one shouldnt be there either

Flying-loulou commented:
I'm not the one that put it there though, and it's not in any torch map folder either
Should I let it as is or should I move both my voidsuit and the explorer one in the mentionned files ?

babydoll commented:
You only need to handle the things you're adding now

Flying-loulou commented:
yes but in this case : what references to torch-specific concepts do you want me to scrub ? (since the onlyone I see is the explorer suit).

babydoll commented:
EC

Flying-loulou committed:
Moves the files to the relevant folders.

Flying-loulou commented:
Should (hopefully) be good now.

Flying-loulou committed:
Corrects the Travis error

Flying-loulou commented:
Fixed Travis telling me to kindly get fucked. Apparently was an issue with the fact that the suit allowed you to carry a health analyzer on the suit storage slot (I just removed this possiblity since BOs don't really need it anyway).

babydoll reviewed (approved):
seems ok

Flying-loulou commented:
Since it's approved, if this could be merged... (I need to clean my repo before doing anything else but I can't as long as this PR is open)
